ra_aid/__main__.py

```
@@ -206,45 +252,66 @@ def main():
+                logger.warning(f"Failed to get project info: {e}")
+                formatted_project_info = ""
+
             # Get initial request from user
-            initial_request = ask_human.invoke({"question": "What would you like help with?"})
+            initial_request = ask_human.invoke(
+                {"question": "What would you like help with?"}
+            )
 
             # Get working directory and current date
             working_directory = os.getcwd()
             current_date = datetime.now().strftime("%Y-%m-%d")
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)